modules/core/src/main/java/org/apache/ignite/internal/client/thin/TcpClientChannel.java

```diff
@@ -807,7 +809,7 @@ else if (!supportedVers.contains(srvVer) ||
                 if (e instanceof IOException)
                     err = handleIOError((IOException)e);
```

Comment:
Perhaps `handleIOError` should be fixed too. It uses `sock.toString()`, but sock it's a `ClientConnection` and doesn't implement `toString`.

Reply:
Yeah.. I fixed this message